stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt

```diff
@@ -84,21 +84,26 @@ public fun Channel.isOneToOne(currentUser: User?): Boolean {
  * @return The text that represent the member status of the channel.
  */
 public fun Channel.getMembersStatusText(context: Context, currentUser: User?): String {
-    val otherMembers = members.filter { it.user.id != currentUser?.id }
-
     return when {
-        otherMembers.isEmpty() -> ""
-        isOneToOne(currentUser) -> otherMembers.first()
+        isOneToOne(currentUser) -> members.first { it.user.id != currentUser?.id }
             .user
             .getLastSeenText(context)
         else -> {
-            val count = otherMembers.count()
-            context.resources.getQuantityString(
-                R.plurals.stream_compose_channel_members,
-                count,
-                count,
-                otherMembers.count { it.user.online }
+            val memberCountString = context.resources.getQuantityString(
+                R.plurals.stream_compose_member_count,
+                memberCount,
+                memberCount
             )
+
+            return if (watcherCount > 0) {
```

Comment:
Discussed with the backend team, and they claim that we can rely on the `watcherCount` field, which actually represents online users (not only the users who are watching the channel, but also connected to the WS). We might have some inconsistencies for some time as we had permissions disabled in the dashboard.

Reply:
I've seen the discussion - I think if they stand behind it, then that's completely okay. Especially if we can show the correct number of people that are in the channel and not just the ones we paginated for